docs/http.md

```diff
@@ -566,10 +569,6 @@ A `Lane` represents a turn lane at the corresponding turn location.
 #### Example
 
 ```json
```

Comment:
I think it is fine to have both. This would document the structure of the `Lane` object.